library/src/scala/internal/quoted/Expr.scala

```diff
@@ -55,4 +55,16 @@ object Expr {
     new Matcher.QuoteMatcher[qctx.type].termMatch(scrutineeExpr.unseal, patternExpr.unseal, hasTypeSplices).asInstanceOf[Option[Tup]]
   }
 
+  /** Returns a null expresssion equivalent to `'{null}` */
+  def `null`: QuoteContext ?=> quoted.Expr[Null] = qctx ?=> {
+    import qctx.tasty._
+    Literal(Constant(null)).seal.asInstanceOf[quoted.Expr[Null]]
+  }
+
+  /** Returns a unit expresssion equivalent to `'{}` or `'{()}` */
+  def Unit: QuoteContext ?=> quoted.Expr[Unit] = qctx ?=> {
+    import qctx.tasty._
+    Literal(Constant(())).seal.asInstanceOf[quoted.Expr[Unit]]
+  }
```

Comment:
It seems both can be `val` instead of `def`?

Reply:
They could but they would not get the optimization where the context parameter is passed directly as one of the method parameters and the closure is not created. We effectively call the following method at runtime (not closure creation needed):
```scala
def Unit(using q: QuoteContext): quoted.Expr[Unit] = {
  import qctx.tasty._
  Literal(Constant(())).seal.asInstanceOf[quoted.Expr[Unit]]
}
```

Reply:
Good point, maybe we can write it directly, instead of the original form?

Reply:
It is like this to align with the quote internal encoding to simplify the transformation.

Reply:
I will consider changing the encoding but this is a larger change that will take some time to accomplish.

Reply:
Thanks for the clarification, it makes sense.